src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -350,14 +352,24 @@ const WorkflowView = _.flow(
     })
   }
 
+  /** Returns the href link for the specified article. */
+  getSupportLink(article) { return `https://support.terra.bio/hc/en-us/articles/${article}` }
```

Comment:
TOL: I like this function - looking through the code this could be generally useful as we use the article numbers a lot. It would be nice to create a mapping of numbers to titles and use a function like this - something like: 

```getSupportLink(article.OOM_RETRY)```

Reply:
Would you suggest then that `getSupportLink` should be in a different file so that it can be reused across other pages? If so, can you point me to an appropriate file?

If `getSupportLink` moves, I'm not sure if it would make sense to put the mapping of titles to numbers in the same location, or keep mappings in the individual page files so they are more closely related to the code referencing them.

Reply:
I added a [ticket ](https://broadworkbench.atlassian.net/secure/RapidBoard.jspa?rapidView=110&projectKey=TUB&modal=detail&selectedIssue=TUB-4) for this improvement linking to this line / diff. 

You can leave it here for now.